src/Components/Clock.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
```

Comment:
those are the Props of the `Clock` component, so we can name it just `name`

Reply:
✅